okhttp/src/main/java/com/squareup/okhttp/RequestBody.java

```diff
@@ -74,6 +74,26 @@ public static RequestBody create(final MediaType contentType, final byte[] conte
     };
   }
 
+  /** Returns a new request body that transmits {@code content}. */
```

Comment:
Can you fix the overload on line 60 to call through to this? No sense in having more code than we need.